CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/startup/ConstellationLAFSettings.java

```diff
@@ -0,0 +1,502 @@
+                if (displayerClass != null) {
+                    // get Reflection objects before updating any UI settings
+                    final Field colsReady = displayerClass.getDeclaredField("colorsReady"); //NOSONAR
+                    Class<?>[] argList = null;
+                    final Method initCols = displayerClass.getDeclaredMethod("initColors", argList);
+
+                    // Update the UIManager with settings appropriate for Windows (XP) LAF
+                    UIManager.getDefaults().put("TabbedPane.highlight", activeMidSectionBlue);
+                    UIManager.getDefaults().put("tab_focus_fill_bright", selectedUpperLightBlue);
+                    UIManager.getDefaults().put("tab_focus_fill_dark", selectedLowerDarkBlue);
+                    UIManager.getDefaults().put("tab_unsel_fill_bright", unselectedUpperGray);
+                    UIManager.getDefaults().put("tab_unsel_fill_dark", unselectedLowerGreyBlue);
+                    // Put a light blue highlight on the selected tab
+                    final Color activeLighterMidBlue = new Color(180, 220, 255);
+                    UIManager.getDefaults().put("tab_highlight_header", activeLighterMidBlue);
+                    UIManager.getDefaults().put("tab_highlight_header_fill", activeLighterMidBlue);
+                    // The following settings are used in the WinXPEditorTabCellRenderer
+                    UIManager.getDefaults().put("TabbedPane.selectionIndicator", activeLighterMidBlue);
+                    UIManager.getDefaults().put("tab_sel_fill_bright", selectedUpperLightBlue);
+                    UIManager.getDefaults().put("tab_sel_fill_dark", selectedLowerDarkBlue);
+
+                    // reset static field "colorsReady" value to false
+                    colsReady.setAccessible(true); //NOSONAR
+                    colsReady.setBoolean(displayerClass, false); //NOSONAR    
+                    
+                    // re-run class method "initColors" to load the updated colors
+                    initCols.setAccessible(true); //NOSONAR
+                    initCols.invoke(displayerClass);
+                } else {
+                    // Unable to apply changes to Windows LAF
+                    // Possibly caused by a change in a newer NetBeans version
+                    LOGGER.info(" >>> Unable to apply Tab coloring to Windows LAF : neither Windows8VectorViewTabDisplayerUI nor WinXPViewTabDisplayerUI are defined :");
+                    ouputUIDefaultValues(null, null);
+                }
+            }
+        } catch (final IllegalAccessException | IllegalArgumentException | InvocationTargetException |
+                NoSuchMethodException | NoSuchFieldException | SecurityException e) {
+            // An exception here would indicate a change in the internal LAF 
+            // packages provided by NetBeans
+            final String errorMessage = " >>> Error applying Windows LaF colors : " + e.toString();
```

Comment:
can you change `e.toString()` to `e.getLocalisedMessage()`?

Reply:
Will not do. The getMessage() and getLocalisedMessage() functions don't provide as much detail. In particular, they don't include the type of Exception caught in the output.

Reply:
Possibly not. On the other hand, to my understanding the info on the exception type should be handled when passing the throwable to the log call after this so you may get that information anyway without needing to call `toString()` specifically.
Note though I made the suggestion as I wasn't sure exactly what `toString()` would return as sometimes that function doesn't always return in the desired format

Reply:
Ok, code has been changed. Now that we're doing a Level.SEVERE log, the full stack trace is being included in the log, so there's no need for the extra details in the message string.